packages/core/src/base-axis-chart.ts

```diff
@@ -176,6 +178,10 @@ export class BaseAxisChart extends BaseChart {
 			this.repositionXAxisTitle();
 		}
 
+		if (this.innerWrap.select(".axis-label.y").nodes().length > 0 && this.options.scales.y.title) {
+			this.repositionYAxisTitle();
```

Comment:
we should do an `||` here. In case you had no title, but you decide to add one with an update. Would a change as simple as that do the trick?

Reply:
Sure, I will change the x axis title in that case as well since it is doing the same thing